### PR TITLE
feat: Add count number of cwt peaks feature

### PR DESCRIPTION
**Pull request by MathieuCayssol**

### CODE REVIEW

**Comment by MathieuCayssol** [comment]
Should I implement `number_cwt_peaks` given that other features involving Continuous wavelet transform are using numpy and are `scipy` based?

File: functime/feature_extraction/tsfresh.py [hunk 1]
```diff
@@ -1017,6 +1044,44 @@ def percent_recoccuring_values(x: TIME_SERIES_T) -> float:
     count = x.unique_counts()
     return count.filter(count > 1).len() / x.n_unique()
 
+def number_peaks(x: pl.Expr, n: int):
```
Comment (topher-lo) [inline functime/feature_extraction/tsfresh.py:1047]:
```suggestion
def number_peaks(x: TIME_SERIES_T, support: int) -> int:
```

File: functime/feature_extraction/tsfresh.py [hunk 2]
```diff
@@ -1017,6 +1044,44 @@ def percent_recoccuring_values(x: TIME_SERIES_T) -> float:
...
+
+    4 is still the highest value. Here, 4 is not a peak of support 3 because 13 is the 3th neighbour to the right of 4
+    and its bigger than 4.
+
+    Parameters
+    ----------
+    x : pl.Expr | pl.Series
+        Input time-series.
+
+    n : int
```
Comment (topher-lo) [inline functime/feature_extraction/tsfresh.py:1069]:
```suggestion
    support : int
```

File: functime/feature_extraction/tsfresh.py [hunk 3]
```diff
@@ -1017,6 +1044,44 @@ def percent_recoccuring_values(x: TIME_SERIES_T) -> float:
...
+        Support of the peak
+    Returns
+    -------
+    float
+    """
+    res = None
+    for i in range(1, n+1):
```
Comment (topher-lo) [inline functime/feature_extraction/tsfresh.py:1076]:
```suggestion
    for i in range(1, support +1):
```

**Review by topher-lo — APPROVED** [review]

**Pull request merged**